apps/alpha-liquidator/src/config.ts

```diff
@@ -104,6 +104,11 @@ let envSchema = z.object({
     }
   }),
   WS_ENDPOINT: z.string().url().optional(),
+  JUP_SWAP_BASE_URL: z
+    .string()
+    .url()
+    .default("https://public.jupiterapi.com")
```

Comment:
Lets keep the default api as `https://quote-api.jup.ag/v6` as its mentioned in the Jup docs
https://station.jup.ag/docs/apis/swap-api

Reply:
Makes sense, will update now.